tutorials/stats-sensor-space/plot_stats_cluster_1samp_test_time_frequency.py

```diff
@@ -75,20 +72,47 @@
 tfr_epochs.apply_baseline(mode='logratio', baseline=(-.100, 0))
 
 # Crop in time to keep only what is between 0 and 400 ms
-evoked.crop(0., 0.4)
-tfr_epochs.crop(0., 0.4)
+evoked.crop(-0.1, 0.4)
+tfr_epochs.crop(-0.1, 0.4)
 
-epochs_power = tfr_epochs.data[:, 0, :, :]  # take the 1 channel
+epochs_power = tfr_epochs.data
+
+###############################################################################
+# Define connectivity for statistics
```

Comment:
I must say that the naming for adjacency is a bit unfortunate in mne. I was a few times in a situation when I had to search online for mne function that finds adjacency. Because it has connectivity in its name finding it was a bit difficult: I was getting lots of results related to actual connectivity (coherence, phase slope index etc.) but not the function I needed.